--- deepspeed/runtime/utils.py	2024-08-14 03:34:33.663726733 +0800
+++ a.py	2024-07-19 11:14:42.000000000 +0800
@@ -727,9 +727,9 @@
 
     get_accelerator().synchronize()
 
-    if reset_max:
-        get_accelerator().reset_max_memory_cached()
-        get_accelerator().reset_max_memory_allocated()
+    #if reset_max:
+    #    get_accelerator().reset_max_memory_cached()
+    #    get_accelerator().reset_max_memory_allocated()
 
     new_alloced = get_accelerator().memory_allocated()
     new_cached = get_accelerator().memory_cached()
@@ -765,6 +765,7 @@
 
 def empty_cache():
     get_accelerator().empty_cache()
+    return
     get_accelerator().reset_peak_memory_stats()
 
 
@@ -789,7 +790,7 @@
     logger.info(f'CPU Virtual Memory:  used = {used_GB} GB, percent = {vm_stats.percent}%')
 
     # get the peak memory to report correct data, so reset the counter for the next call
-    get_accelerator().reset_peak_memory_stats()
+    # get_accelerator().reset_peak_memory_stats()
 
 
 def call_to_str(base, *args, **kwargs):
